tests/fast-integration/skr-test/README.md

```diff
@@ -2,16 +2,44 @@
 
 This test covers SKR (SAP Kyma Runtime).
 
-## File Structure consists of the following parts:
+## File structure
 - **provision** folder contains the scripts for provisioning and de-provisioning the SKR cluster using KEB client
-- **oidc** folder contains the OIDC related tests
+- **oidc** folder contains the OIDC-related tests
 - **commerce-mock** folder consists of two scripts:
-  1. **index.js** contains the commerce mock tests suite
-  2. **prep.js** can be called in stand-alone mode for commerce mock resources preparation
+  - **index.js** contains the commerce mock tests suite
+  - **prep.js** can be called in stand-alone mode for CommerceMock resources preparation
```

Comment:
Our [styleguide](https://kyma-project.io/community/guidelines/content-guidelines/02-style-and-terminology#capitalized-terms) says we avoid CamelCase and use natural language instead.

```suggestion
  - **prep.js** can be called in stand-alone mode for Commerce Mock resources preparation
```

Reply:
updated